### PR TITLE
BUG-101501 new certificate generated for ambari on master node with public ip as host

### PR DESCRIPTION
**Pull request by attilapalfi92**
- separate configs for the two ssl certs (443 and 9443) for ambari
- script added to generate new cert for the user facing port (443) for ambari

Closes #BUG-101501

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/top.sls [hunk 1]
```diff
@@ -53,6 +53,9 @@ base:
     - prometheus.server
     - ambari.server
     - jdbc.connectors
+{% if not salt['file.directory_exists']('/yarn-private') %}  # FIXME (BUG-92637): must be disabled for YCloud
```
Comment (keyki) [inline orchestrator-salt/src/main/resources/salt/salt/top.sls]: This should be moved to server_start to support gateway recovery.

File: orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls [hunk 2]
```diff
@@ -18,6 +18,10 @@ ambari-start-postgresql:
   file.managed:
     - source: salt://ambari/systemd/ambari-server.service
 
+{% if not salt['file.directory_exists']('/yarn-private') %}  # FIXME (BUG-92637): must be disabled for YCloud
```
Comment (keyki) [inline orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls]: Wouldnt it be better at 'roles:ambari_server': in the top file?

Comment (keyki) [reply orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls]: oh, sorry :) thanks

File: orchestrator-salt/src/main/resources/salt/salt/top.sls (hunk 1 above)
Comment (attilapalfi92) [reply orchestrator-salt/src/main/resources/salt/salt/top.sls]: Moved it.

File: orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls (hunk 2 above)
Comment (attilapalfi92) [reply orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls]: checked it, its working well

File: orchestrator-salt/src/main/resources/salt/salt/nginx/init.sls [hunk 3]
```diff
@@ -1,15 +1,54 @@
...
+
+add_nginx_ycloud_reload_script:
+  file.managed:
+    - name: /opt/salt/scripts/reload-nginx-ycloud.sh
+    - source: salt://nginx/scripts/reload-nginx-ycloud.sh
+    - makedirs: True
+    - template: jinja
+    - mode: 755
+
+run_nginx_ycloud_reload_script:
```
Comment (attilapalfi92) [inline orchestrator-salt/src/main/resources/salt/salt/nginx/init.sls]: @keyki you meant something like this?

**Pull request merged**